samples/Rich-Cards-Bot/README.md

```diff
@@ -28,12 +28,7 @@ git clone https://github.com/Microsoft/botbuilder-python.git
 
 ### Connect to bot using Bot Framework Emulator **V4**
```

Comment:
We've also been removing the bolding around "V4" in the botbuilder-samples repo 😄